apps/web/src/app/[locale]/challenge/_components/comments/index.tsx

```diff
@@ -79,9 +83,11 @@ export function Comments({ preselectedCommentMetadata, rootId, type, expanded =
           className={clsx(
             'custom-scrollable-element flex flex-col overscroll-contain duration-300',
             {
-              'h-64 pb-4 md:h-[calc(100vh_-_164px)]': showComments,
               'h-0 overflow-y-hidden': !showComments,
-              'overflow-y-auto': showComments && (data?.comments.length ?? 0) > 0,
+              'h-64 pb-4 lg:h-[calc(100vh_-_164px)]': showComments && !isTrackTitleVisible, // no title
+              'h-[calc(256px_-_46px)] pb-4 lg:h-[calc(100vh_-_210px)]':
+                showComments && isTrackTitleVisible, // with title
+              'overflow-y-auto ': showComments && (data?.comments.length ?? 0) > 0,
```

Comment:
thought: This looks a little dense, @me-imfhd.It might be brittle to keep in-sync. For example, that `210px` needs to be maintained as the exact height of the title and header.

I'm wondering if there is a simpler way. If this element is not meant to be `100vh`, I wonder if we should not be using `100vh` in the calculation. Instead, maybe a parent/ancestor can be `100vh` and the comments drawer here can instead use `height: 100%` to fill the size of it's parent/ancestor.

Reply:
hey @ayrock-dev thanks for reviewing, i tried the simpler way but it didn't work out as i expected, reason was the way components were written or just maybe i am not that skilled yet...
and the other thing that
yeah that 210px is not dynamic, but for the time being since everything was static so the code was quite working for everything so i didn't wanted to increase the code logic thus i stick with this, but maybe this can be written better...

By the way, my reason to write h-[calc(256px_-_46px)] this instead of h-[210px] was that h-64 is equal to 256px and thus we know reducing 46px from h-64 will make it work, maybe it was not good idea but that was my resolve.
But i can just write h-[210px] if its asked.